media/css/products/relay/components/matrix.scss

```diff
@@ -185,17 +186,26 @@ $brand-theme: 'firefox';
     grid-area: price;
 }
 
-.c-matrix-footer-savings {
+.c-matrix-footer-vpn-savings {
+    line-height: 1;
+    font-size: 14px;
```

Comment:
This could be `@include text-body-sm;` or at least `@include font-size(14px)` so it'll output rems.